### PR TITLE
Add KML generator endpoint & example KML files

### PR DESCRIPTION
**Pull request by Andrew-Dickinson**
Adds a kml generator, to reproduce the behavior of the apps-script hosted one. I reverse engineered the apps script logic and checked against the output files, so this should produce KML files that are isomorphic to the originals. Once it's up we can send one to Olivier for verification and feedback

Screenshot 2024-02-08 at 00 25 48

### CODE REVIEW

File: src/meshapi/views/geography.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    "Staten Island": "Staten Island",
+    None: "Other",
+}
+
+
+@api_view(["GET"])
+@permission_classes([permissions.AllowAny])
+def map_kml(request):
+    kml_root = kml.KML()
+    ns = "{http://www.opengis.net/kml/2.2}"
```
Comment (WillNilges) [inline src/meshapi/views/geography.py:34]: Just a thought, we should document all our external dependencies somewhere 🤔

Comment (Andrew-Dickinson) [reply src/meshapi/views/geography.py:34]: Sure, but this isn’t a dependency. This is just a namespace for the XML schema that KML uses. We don’t call this at runtime

Comment (WillNilges) [reply src/meshapi/views/geography.py:34]: oh, lol

**Review by WillNilges — APPROVED** [review]
Wow, this is sick. Should we put a frontend on meshforms?

**Comment by Andrew-Dickinson** [comment]
If by frontend you mean "a link to the KML download" then sure haha

**Comment by Andrew-Dickinson** [comment]
Whoops this was merged to the wrong place. Cherry picked this onto main via f1126795c1fbe375ae29fb403016eb3955f4fc09

**Pull request merged**